web/ui/react-app/test/Status.test.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Status } from '../src/pages';
+import { Alert } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as useFetch from '../src/hooks/useFetches';
+import toJson from 'enzyme-to-json'
+
+describe('Status', () => {
+  afterEach(() => jest.restoreAllMocks());
+  it('should render spinner while waiting data', () => {
+    const wrapper = shallow(<Status />);
+    expect(wrapper.find(FontAwesomeIcon)).toHaveLength(1);
+  });
+  it('should render Alert on error', () => {
+    (useFetch as any).default = jest.fn().mockImplementation(() => ({ error: new Error('foo') }))
+    const wrapper = shallow(<Status />);
+    expect(wrapper.find(Alert)).toHaveLength(1);
+  });
+  it('should fetch proper API endpoints', () => {
+    const useFetchSpy = jest.spyOn(useFetch, 'default');
+    shallow(<Status />);
+    expect(useFetchSpy).toHaveBeenCalledWith(['../api/v1/status/runtimeinfo', '../api/v1/status/buildinfo', '../api/v1/alertmanagers']);
+  });
+  describe('Snapshot testing', () => {
+    const response = [
+      {
+        StartTime: '2019-10-30T22:03:23.247913868+02:00',
```

Comment:
It probably doesn't matter for the tests, but in the actual API response, these fields are differently cased now.